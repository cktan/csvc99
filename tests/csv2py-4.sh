# Test Case : CSV with Line Breaks in Data 
../csv2py in/csv2py-4.csv 
