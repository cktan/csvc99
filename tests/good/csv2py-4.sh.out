[
	['name','description'],
	['John Doe','Line 1\nLine 2'],
	['Jane','Single line']
]

